src/Bedrock.Framework.Experimental/Protocols/Tls/BulkCiphers/AdditionalInfo.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Bedrock.Framework.Experimental.Infrastructure;
+using System.Net;
+using System.Runtime.InteropServices;
+
+namespace Bedrock.Framework.Experimental.Protocols.Tls.BulkCiphers
+{
+    [StructLayout(LayoutKind.Sequential, Pack = 1)]
+    internal struct AdditionalInfo
```

Comment:
Mutable struct? Why?

Reply:
There is a sequence number on this we increased for each message sent. We then feed this into BulkCipher as additional information used during the encryption for AES etc. Also by making the methods convert to the correct bitness in the set it means we can directly blit the struct (by using a pointer)